dev/io.openliberty.http.monitor_fat/bnd.bnd

```diff
@@ -20,6 +20,39 @@ src: \
 
 fat.project: true
 
+fat.minimum.java.level: 11
```

Comment:
JEE 7 uses java 8.  Are the tests require Java 11?

Reply:
This follows what the rest of the MP telemetry FATS are using.
See: https://github.com/OpenLiberty/open-liberty/blob/integration/dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/bnd.bnd#L16

Reply:
Remember that Yasmin is adding Java 8 support to MP Tel 2.0 ... and so some of the FAT buckets might be changing from Java 11 back down to Java 8.